pkg/operator/allocations/set_allocation_delay.go

```diff
@@ -168,6 +169,11 @@ func readAndValidateAllocationDelayConfig(c *cli.Context, logger logging.Logger)
 	broadcast := c.Bool(flags.BroadcastFlag.Name)
 	operatorAddress := c.String(flags.OperatorAddressFlag.Name)
 
+	callerAddress := c.String(flags.CallerAddressFlag.Name)
+	if common.IsEmptyString(callerAddress) {
```

Comment:
Nit: might be helpful to add a info log here about this replacement.

Reply:
oh that's true. good catch